src/js/Components/RacingBoards.js

```diff
@@ -0,0 +1,80 @@
+import Events from '../Constants/Events.js'
+import { Congratulation } from '../Constants/Message.js'
+import Component from '../Core/Component.js'
+import { SET_MANUAL } from '../modules/actions.js'
+import { displayWinners, moveCars, setManual } from '../modules/creator.js'
+import { store } from '../modules/store.js'
+import { getRandomInt } from '../utils/random.js'
+
+export default class RacingBoards extends Component {
+  constructor(target) {
+    super(target)
+  }
+
+  setEvent(target) {}
+
+  dispatchMoveCars() {
+    setTimeout(() => {
+      store.dispatch(moveCars())
+    }, 1000)
+  }
+
+  template() {
+    const { cars, totalAttemps, remainAttemps, manual } = store.getState()
+
+    const gameEnded = remainAttemps === 0
+
+    if (totalAttemps <= 0) return ''
+
+    if (!manual) {
+      if (remainAttemps > 0) {
+        this.dispatchMoveCars()
+      }
+    }
```

Comment:
```suggestion
    if (!manual && remainAttemps > 0) {
        this.dispatchMoveCars()
      }
```
이렇게 조건을 바꿔줘도 괜찮을꺼 같아요